eng/pipelines/templates/jobs/sdk-build.yml

```diff
@@ -12,6 +12,7 @@ parameters:
   publishRetryConfig: false
   publishXunitResults: false
   enableSbom: true
+  timeoutInMinutes: 150
```

Comment:
separately, we do need to figure out why legs sometimes are taking more than 2h. Our ideal target is 1h though that can be challenging to achieve.